lang/ar.yml

```diff
@@ -41,6 +41,7 @@ ar:
     EXTRACLASSA: 'تصميم/تخطيط إضافي'
     EXTRACLASSB: 'فئة إضافية من صفحات الأنماط التعاقبية CSS - افصل بين المتعددة بفضاء'
     FIELDCONFIGURATION: 'مواصفات الحقل'
+    FIELDISREQUIRED: '{name} ضروري'
```

Comment:
Can you please either confirm you're a speaker, or know someone who has looked over this and confirmed it's correct? It is a RTL language to my understanding, but that doesn't mean that this sentence structure is wrong... but I certainly don't know :/

Same goes for all the other languages too I guess...

Reply:
Yeah, great question. I only speak English. Fortunately, these additional lines are a duplicate (in all case). In this file the source is line 23. Is that more agreeable?

Reply:
Yeah neat, that makes perfect sense then :)